test/library/packages/UnitTest/Launcher.chpl

```diff
@@ -15,6 +15,32 @@ module Launcher {
   config const setComm: string = "";
   var comm: string;
 
+  /* Gets the comm */
+  proc getRuntimeComm() throws {
+    var line: string;
+    var checkComm = spawn(["python",CHPL_HOME:string+"/util/chplenv/chpl_comm.py"],
+                        stdout = PIPE);
+    while checkComm.stdout.readline(line) {
+      comm = line.strip();
+    }
+    // setting communication mechanism.
+    if setComm != "" {
+      if comm != "none" {
```

Comment:
Shouldn't `setComm` override `comm` regardless of the value from `comm` or am I confused about the intended behavior of `setComm`? It seems like this logic could be cleaned up a little.

Reply:
I would add a doc-string to `setComm` to clarify the purpose / rules as well.

Reply:
`setComm` will override the `comm`. But if the `comm=none` and user wants to run in `gasnet`, should we allow that?

Reply:
Ah I see. No that shouldn't be necessary right now. Thanks.